test/PooltogetherGovernor.t.sol

```diff
@@ -135,4 +137,162 @@ contract Propose is ProposalTest {
     // Ensure the governorBravo is now the admin of the timelock
     assertEq(timelock.admin(), address(governorBravo));
   }
+
+  ////
+  // Post proposal tests
+  ////
+  function testFuzz_OldGovernorSendsETHAfterProposalIsDefeated(uint128 _amount, address _receiver)
+    public
+  {
+    _assumeReceiver(_receiver);
+
+    // Counter-intuitively, the Governor (not the Timelock) must hold the ETH.
+    // See the deployed GovernorAlpha, line 204:
+    //   https://etherscan.io/address/0xB3a87172F555ae2a2AB79Be60B336D2F7D0187f0#code
+    // The governor transfers ETH to the Timelock in the process of executing
+    // the proposal. The Timelock then just passes that ETH along.
+    vm.deal(address(governorAlpha), _amount);
+
+    uint256 _receiverETHBalance = _receiver.balance;
+    uint256 _governorETHBalance = address(governorAlpha).balance;
+
+    // Defeat the proposal to upgrade the Governor
+    _defeatUpgradeProposal();
+
+    // Create a new proposal to send the ETH.
+    address[] memory _targets = new address[](1);
+    uint256[] memory _values = new uint256[](1);
+    _targets[0] = _receiver;
+    _values[0] = _amount;
+
+    _queueAndVoteAndExecuteProposalWithAlphaGovernor(
+      _targets,
+      _values,
+      new string[](1), // No signature needed for an ETH send.
+      new bytes[](1), // No calldata needed for an ETH send.
+      true // GovernorAlpha is still the Timelock admin.
+    );
+
+    // Ensure the ETH has been transferred to the receiver
+    assertEq(address(governorAlpha).balance, _governorETHBalance - _amount);
+    assertEq(_receiver.balance, _receiverETHBalance + _amount);
+  }
+
+  function testFuzz_OldGovernorCannotSendETHAfterProposalSucceeds(
+    uint256 _amount,
+    address _receiver
+  ) public {
+    _assumeReceiver(_receiver);
+
+    // Counter-intuitively, the Governor must hold the ETH, not the Timelock.
+    // See the deployed GovernorAlpha, line 227:
+    //   https://etherscan.io/address/0xDbD27635A534A3d3169Ef0498beB56Fb9c937489#code
```

Comment:
Needs to be updated

Reply:
https://github.com/ScopeLift/pooltogether-governor-upgrade/pull/12/commits/e559c02640193e65b0305cd4d8fd2ae3c7cf27e4#diff-c21c7f9958e9aa1f065be9cab617cf6ffda75c3944410c62779e3eefeb2a2d4cR189